### PR TITLE
Feature/widerruf pages

### PR DESCRIPTION
**Pull request by GeBon22**
I added the necessary pages for the widerruf content. There is of course more to do on this matter - there is going to be a follow up user storie taking care of the missing/unfinished/unstyled parts. 

For now please consider this a first draft.

### CODE REVIEW

File: pages/widerruf/frist.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+        <StyledBreadcrumb>
+          <StyledLink href="/widerruf">/Widerruf und Retoure</StyledLink>/Wie
+          lange kann ich widerrufen?
+        </StyledBreadcrumb>
+        <StyledArticle>
+          <Styledh3>Wie lange kann ich widerrufen?</Styledh3>
+          Der Widerruf muss binnen 14 Tagen nach Erhalt der Ware erklärt werden.
+          Das bedeutet nicht, dass binnen dieser Zeit auch die Ware zurück an
+          den Verkäufer gesendet werden muss.
+          <br />
```
Comment (stefankorder) [inline pages/widerruf/frist.js]: Maybe there is a better solution instead of using `<br>`.

**Review by stefankorder — CHANGES_REQUESTED** [review]
Looks good so far! Please take a look at my comment.

File: pages/widerruf/frist.js (hunk 1 above)
Comment (GeBon22) [reply pages/widerruf/frist.js]: check

File: pages/widerruf/frist.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    den Verkäufer gesendet werden muss. Die Ware muss innerhalb weiterer 14 Tage ab Erklärung des Widerrufs
+    versendet werden. 
+    Achte auf die Widerrufsregelungen im Shop des Verkäufers. Dort ist
+    meist der Zeitraum für die Rücksendung der Ware explizit geregelt.
+    Meistens liegt dieser bei 30 Tagen.
+
+    TLDR: binnen 14 Tagen erklären, je nach Widerrufsregelung des
+    Verkäufers zwischen 14 und 30 Tagen ab Erklärung zurück senden
+      `}
+        </StyledArticle>
```
Comment (actyralf) [inline pages/widerruf/frist.js:27]: It would be better to group your content in paragraphs (`<p>`)

**Review by actyralf — CHANGES_REQUESTED** [review]
One last thing...

File: pages/widerruf/frist.js (hunk 2 above)
Comment (GeBon22) [reply pages/widerruf/frist.js:27]: ok :-)

**Pull request merged**